tests/unit/backend/layers/api/test_portal_api.py

```diff
@@ -1765,7 +1765,7 @@ def test__get_all_datasets_for_user_index(self):
                 # append the dataset_id to the list of dataset ids
                 dataset_ids.append(dataset["dataset_id"])
 
-        self.assertEqual(len(dataset_ids), 6)
+        self.assertEqual(len(dataset_ids), 8)
```

Comment:
what's going on here? 🔍 🤔

Reply:
This test counts how many dataset assets appear when fetching dataset metadata. Previously, our mock did not include RAW_H5AD. Now it does, which reflects the state of real datasets